.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "nbgv": {
+      "version": "3.4.194",
```

Comment:
Do can this be a wildcard to 3.x or versionless?

Reply:
> Invalid manifest file. Path C:\repos\aaronla-ms\reproducible-builds\.config\dotnet-tools.json:
>        In package nbgv:
>                Version 3.* is invalid.

Doesn't appear to be supported. I'll ask around.

Reply:
I'm recommending we keep the fixed version for now. If I hear back any supported alternatives, will send a new PR.